lighty-applications/lighty-rnc-app-aggregator/lighty-rnc-app-helm/helm/lighty-rnc-app-helm/values.yaml

```diff
@@ -69,6 +69,14 @@ ingress:
   exposeManagement: false
   managementHost: "management.lighty.io"
 
+deploymentSecurity:
+  usePodSecurity: true
+  useContainerSecurity: true
+  # User with UID 1000 is set in RNC docker image
+  runAsUser: 1000
+  # Description for possible capabilities https://github.com/torvalds/linux/blob/master/include/uapi/linux/capability.h
+  useCapabilities: ["NET_BIND_SERVICE"]
```

Comment:
We are just binding 8888 - why we need capability?

Reply:
Removed